charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -71,10 +85,38 @@ subjects:
   apiGroup: rbac.authorization.k8s.io
 roleRef:
   kind: ClusterRole
-  name:  pp-plugins-scheduler
+  name:  system:pp-plugins-scheduler
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name:  pp-plugins-scheduler-auth-delegator
+subjects:
+- kind: User
+  name: system:kube-scheduler
+  namespace: {{ .Release.Namespace }}
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name:  pp-plugins-scheduler:system:auth-delegator
```

Comment:
Where is this cluster role `pp-plugins-scheduler:system:auth-delegator`?

`pp-plugins-scheduler:system:auth-delegator` in this file is a `ClusterRoleBinding`

Reply:
This comment is still outstanding. I don't see a cluster role with that name.

Reply:
it has been removed from the config and helm files but need to remove it from here as well